TYPO3.Flow/Classes/TYPO3/Flow/Package/PackageInterface.php

```diff
@@ -152,7 +154,7 @@ public function getDocumentationPath();
      * Returns the available documentations for this package
      *
      * @return array Array of \TYPO3\Flow\Package\Documentation
-     * @api
+     * @deprecated To be removed in Flow 4.0
```

Comment:
IMHO that's up to your application to decide. It's pretty hardcoded in here, so if you want to keep these paths you can hardcode it in your app. Alternatively I would suggest to extend the composer.json with the information (extra.paths.documentation eg.)